package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mobx",
-  "version": "5.14.1",
```

Comment:
Uh, please remove the bump, it's done automatically in the release process.

Reply:
Apologies, fixed.